ReadMe.md

```diff
@@ -30,57 +25,96 @@ The Aiko Engine MP provides ...
 - Supports [LCA2017 IoTuz project](http://www.openhardwareconf.org/wiki/OHC2017)
 - Tested on a range of ESP32 development boards
 
-<a name="installation" />
+## Installation
+
+Installation is a three stage process:
+
+1. Setup the correct tooling on your workstation
+2. Download and install MicroPython onto your ESP32 development board
+3. Install the Aiko software onto your ESP32 development board.
+
+**Notes**:
 
-Installation
--------------
+* If you are using Linux or macOS then you will be using a Posix shell (e.g. [Bash](https://www.gnu.org/software/bash/) or [zsh](http://zsh.sourceforge.net/))
+* If you are on Windows then we assume you are using [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/overview) on Windows 10.
+
+### 1. Setup
+
+- Make sure you have a current version of [Python3](https://www.python.org/) available on your workstation
+
+- Download the Aiko software code from https://github.com/geekscape/aiko_engine_mp
 
-- Download the source code from <https://github.com/geekscape/aiko_engine_mp>
 ```
     git clone https://github.com/geekscape/aiko_engine_mp
+```
+**Note**: If you don't have Git installed you can download a zip file from https://github.com/geekscape/aiko_engine_mp and unpack it
+
+
+```
     cd aiko_engine_mp
 ```
-- Ensure you have a
-  [compatible hardware board](https://github.com/micropython/micropython/wiki/Boards-Summary) or
-  [compatible microPython port](https://github.com/micropython/micropython/tree/master/ports)
-- Make sure you have a Python
-  [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/#lower-level-virtualenv) set-up, including
-  [virtualenvwrapper](http://docs.python-guide.org/en/latest/dev/virtualenvs/#virtualenvwrapper)
-- Create `mkvirtualenv micropython` and work on the new virtual environment
+
+**Note**: All commands below assume that `.../aiko_engine_mp` is your current directory.
+
+
+- Setup a [virtual Python environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) for your badge project. e.g.
+
 ```
-    workon micropython
+python3 -m venv .venv
 ```
-- Install mpfshell
+
+- Make sure your Python virtual environment is active. **Note**: Do this every time a new terminal session is started
+
+```
+. .venv/bin/activate # Linux and macOS
+. .venv\Scripts\activate.ps1 # Windows
+```
```

Comment:
It's a little fiddly, but there'll be at least one indent level at which you can put the fenced code block inside the list item instead of stopping the list and starting a new one afterward.